lib/libzpool/Makefile.am

```diff
@@ -239,3 +242,4 @@ vdev_raidz_math_powerpc_altivec.l$(OBJEXT): CFLAGS += -maltivec
 endif
 
 include $(top_srcdir)/config/CppCheck.am
+AM_CFLAGS += -no-suppress
```

Comment:
Stupid question maybe but is this really supposed to be here twice? (I'm fully prepared for Autohell Weirdness™ to mean Yes, but)

Reply:
Absolutely not, I just echoed it into the end of every file and then moved it up to somewhere reasonable, but I had already put it in this file because it was where I hit the problem initially, so I presumably just didn't go edit it at all.